cpanfile

```diff
@@ -8,6 +8,8 @@ requires 'JSON::MaybeXS', '>= 1.003008';
 requires 'List::Util';
 requires 'Scalar::Util';
 requires 'Future', '>= 0.37';
+requires 'Digest::Keccak', '>= 0.05';
+requires 'Path::Tiny';
```

Comment:
is this package is being used somewhere ?

Reply:
https://github.com/binary-com/perl-Ethereum-RPC-Client/blob/master/lib/Ethereum/RPC/Contract/Helper/ImportHelper.pm#L33